bvm/ballerina-runtime/README.md

```diff
@@ -0,0 +1,256 @@
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Construct**                            | **Description**                                                                                                                                                                                                                                                                                                                                                             |
+| io.ballerina.runtime.api.Environment     | Developers can use this as the first argument of an interop method, Ballerina will inject an instance of Environment when calling. That instance can be used to communicate with the currently executing Ballerina runtime.With Environment you can get interop ballerina function name, path parameters, strand id, strand metadata, current module , current runtime etc. |
+| io.ballerina.runtime.api.Future          | This will contain the future value once we call the ballerina method from API asynchrounously.                                                                                                                                                                                                                                                                              |
+| io.ballerina.runtime.api.Module          | Represent Java runtime module.                                                                                                                                                                                                                                                                                                                                              |
+| io.ballerina.runtime.api.PredefinedTypes | Contains Predefined types.                                                                                                                                                                                                                                                                                                                                                  |
+| io.ballerina.runtime.api.Runtime         | You can get an instance of current runtime through an Environment instance. This will contain APIs to call ballerina object methods asynchronously.                                                                                                                                                                                                                         |
```

Comment:
```suggestion
| io.ballerina.runtime.api.Runtime         | An instance of the current runtime can be obtained through an Environment instance. This will contain APIs to call ballerina object methods asynchronously.                                                                                                                                                                                                                         |
```